README.md

```diff
@@ -47,16 +47,26 @@ docker run -p 6379:6379 -p 3000:3000 -it --rm  -v ./data:/data falkordb/falkordb
 Currently, this SDK supports the following LLMs API:
 
 * [OpenAI](https://openai.com/index/openai-api) Recommended model:`gpt-4o`
-* [google](https://makersuite.google.com/app/apikey) Recommended model:`gemini-1.5-flash-001`
-
-Make sure that a `.env` file is present with all required credentials.
+* [Google](https://makersuite.google.com/app/apikey) Recommended model:`gemini-1.5-flash-001`
+* [Azure-OpenAI](https://ai.azure.com) Recommended model:`gpt-4o`
+* [Ollama](https://ollama.com/) Available only to the Q&A step. Recommended models: `llama3`. Ollama models are suitable for cases where the knowledge graph (KG) is predefined.
+  
+    To get started with Ollama:
+    1. Install Ollama from https://ollama.com/
+    2. Pull your preferred model: `ollama pull llama3`

```

Comment:
Why do we have this section here?

If we believe this information is necessary, then let's create a dedicate section: `Getting Started` where each model is covered.